### ISSUE TITLE
Optional file suffixes for split output

### ISSUE THREAD

**Issue by privatezero** [mipops/dvrescue#356]
As per the discussion at recent meeting, we need to think of a list of desirable suffixes for tape files that are split by DV Packager. So far ideas include:
* Time code
* _part1 (_part2, _part3 etc.)
* others??

**Comment by dericed** [comment]
How would timecode be formatted in a filename. For instance `tc=01:53:22;24`, how should the filename expression of that be formed?
Others:
- pts of input (current)
- recording timestamp, but also need to format this
- frame number of input or frame range of input

**Comment by privatezero** [comment]
@dericed I like that - that maps exactly to our naming conventions here are UW! (If there is no _part# we assume it was all one)

**Comment by libbyhopfauf** [comment]
I agree with adding zeros and the name conventions of file_part1-2.mov and file_part2-2.mov (etc)!

**Comment by dericed** [comment]
Hey @iamdamosuzuki @libbyhopfauf @privatezero, it's been a while since discussing, but is the current state working for everyone or should we split up and support a short list of naming strategies?

**Comment by privatezero** [comment]
Current state doesn't have the leading zero, which I agree it probably should have. Also, in my use case at least, would be preferable to not have any `_part` suffix in instances when there is only a single part. Not sure about other's naming schemes, but in ours having a `_part` or `_side` suffix always implies the existence of other parts, and with files labeled `_part01` with no `_part02` this could cause confusion years down the road. Pretty much in line I guess with @libbyhopfauf's suggestion for `_part1-1` `_part2-1` in terms of wanting to avoid ambiguity as well as manual renames after the fact.

(I recognize that this would make the file naming more of a pain in the ass though as it would necessitate either a check/rename after the fact or a call against the XML to determine the number of expected output files with the chosen segmentation rules to happen as part of the file naming)

**Comment by libbyhopfauf** [comment]
I agree with @privatezero that I still think this would be the best option and with the leading zero (I spent too much of one of my day last week check and fixing files to make this consistent). I also agree that not having the `_part` suffix for a single part would be preferable. 

And I hear the issue with having to check the XML for all and renaming for those who want to keep it as part of the name. If this is a common enough need, would there be a way where we could have these both as options and people can pick?

**Comment by privatezero** [comment]
Just wanted to draw attention back to this issue as it still is throwing a (very small but persistent) monkey wrench into our workflows. A majority of our tapes recently are only being packaged into a single file and we have to manually edit each filename post packaging to remove the extraneous `_part1` suffix. I suspect this is a common issue for other users as well
